installer/pom.xml

```diff
@@ -72,6 +72,29 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>3.1.0</version>
```

Comment:
Plugin version should be derived, not specified.  The parent pom has the resource plugin in the plugin management section.  But, the version may need to be updated.  Currently, the maven-resources-plugin is set to 2.6.

Reply:
It works with both 2.6 and 3.1.0, so left the parent pom at 2.6.